app/assets/stylesheets/_typography.scss

```diff
@@ -0,0 +1,23 @@
+// Copied and cut down from node_modules/govuk-frontend/govuk/core/_typography
+@import "lib/govuk-frontend/govuk/settings/all";
+@import "lib/govuk-frontend/govuk/helpers/all";
+@import "lib/govuk-frontend/govuk/tools/all";
+
+$govuk-typography-use-rem: false;
+
+#main-content {
+  padding-top: 0px;
+  p, div {
```

Comment:
Rather than copying from govuk-frontend, you may want to consider enabling global styles: https://github.com/alphagov/govuk-frontend/blob/803da2c92d1a73401ac506b2dc8a393fae940508/src/govuk/settings/_global-styles.scss#L13

Reply:
Having said that, the recommended pattern is to use classes instead: https://design-system.service.gov.uk/get-started/production/